std_msgs/Header header 
float32[] box_classes
float32[] box_bounding_boxes
float32[] confidences
int32 nPredictions
